web/package.json

```diff
@@ -25,10 +25,16 @@
     "@angular/router": "^5.2.11",
     "@angular/service-worker": "^5.2.11",
     "angular2-notifications": "^1.0.2",
+    "bootstrap": "^4.3.1",
     "core-js": "^2.5.7",
     "http-status-codes": "^1.3.0",
     "intl": "^1.2.5",
+    "j": "^1.0.0",
```

Comment:
I'm not sure why do you add this dependency. Do we need it?

Reply:
Actually I didn't edit package.json manually - only through installations.
I tried to run the app without it - successful, no errors.
I also tried to go through whole installation process with my training project - there was no such dependency.
I'm not sure if we need this dependency at all.

Reply:
About this dependency **"j": "^1.0.0"**, other dependencies are needed

Reply:
What command did you use to insulation?

Reply:
First of all, I tried to install bootstrap but failed with an error. It was mentioned in the log that node-sass was not found. So I went through like this:
npm install node-sass --save
npm install bootstrap@4 --save
npm install jquery --save
npm install popper.js --save
npm install tether --save

It was recomended to install "tether" as well so I added it. There is a similar issue:
https://stackoverflow.com/questions/50192628/how-to-install-bootstrap-4-jquery-popper-js-with-angular-6
Our drop-down menu works fine now. I'm not sure if going with ng-bootstrap would've been a better idea though.

Reply:
I see only bootstrap, jquery and popper before. Why did you add other dependencies?

Reply:
Considering installing unnecessary dependencies - please, check out you Slack.